src/input_xml.F90

```diff
@@ -5856,7 +5813,8 @@ subroutine read_ce_cross_sections(libraries, library_dict)
           ! Read nuclide data from HDF5
           file_id = file_open(libraries(i_library) % path, 'r')
           group_id = open_group(file_id, name)
-          call nuclides(i_nuclide) % from_hdf5(group_id)
+          call nuclides(i_nuclide) % from_hdf5(group_id, &
+                                               materials(i) % temperature)
```

Comment:
Because `nuclide_dict` and `already_read` now use a name without a cross section (temperature) identifier, it seems like this would only get hit for the first temperature. Moreover, when nuclides are assigned to materials (up around line 2480), it seems like a second temperature would never get assigned in the first place. Am I misunderstanding something here?